terraso_backend/apps/auth/services.py

```diff
@@ -172,16 +175,33 @@ def verify_unsubscribe_token(self, token):
             raise ValueError("Token is not an unsubscribe token")
         return decoded
 
+    def create_story_map_membership_approve_token(self, membership):
+        user = membership.user
+        return self.create_token(
+            user,
+            extra_payload={
+                "membershipId": str(membership.id),
+                "pendingEmail": membership.pending_email if user is None else None,
+                "approveStoryMapMembership": True,
```

Comment:
question: Is there ever a case when `approveStoryMapMembership` should be set false?

Reply:
Kind of, this was added as part of a fix to add an explicit intent to the tokens, see: https://github.com/techmatters/terraso-backend/pull/777, so some way to look at this is that `approveStoryMapMembership` will be false for all the other generated tokens